xclim/data/fr.json

```
@@ -435,6 +434,13 @@
     "comment": "",
     "abstract": "Nombre de jours de vagues de froid. Une vague de froid se produit lorsque la température journalière minimale est sous un seuil durant au moins un certain nombre de jours consécutifs."
   },
+  "COOL_NIGHT_INDEX": {
+    "long_name": "Indice  de  fraîcheur  des  nuits",
+    "description": "Moyenne {freq:f} de la température journalière minimale en septembre (hémisphère nord) ou en mars (hémisphère sud)",
+    "title": "Indice  de  fraîcheur  des  nuits",
```

```suggestion
    "long_name": "Indice de fraîcheur des nuits",
    "description": "Moyenne {freq:f} de la température journalière minimale en septembre (hémisphère nord) ou en mars (hémisphère sud)",
    "title": "Indice de fraîcheur des nuits",
```
V e r y   s p a c y!

---

A E S T H E T I C S